h5py/h5f.pyx

```diff
@@ -81,6 +82,14 @@ IF HDF5_VERSION >= (1, 10, 1):
     FSPACE_STRATEGY_AGGR = H5F_FSPACE_STRATEGY_AGGR
     FSPACE_STRATEGY_NONE = H5F_FSPACE_STRATEGY_NONE
 
+    # Used in FileID.get_page_buffering_stats()
+    PageBufStats = namedtuple('PageBufferStats', ['meta', 'raw'])
+    MetaStats = namedtuple(
+        'MetaStats', ['accesses', 'hits', 'misses', 'evictions', 'bypasses'])
+    RawStats = namedtuple(
+        'RawStats', ['accesses', 'hits', 'misses', 'evictions', 'bypasses'])
```

Comment:
I think we only need one type here, as they've got the same fields, right?

Reply:
Replaced `MetaStats` and `RawStats` with one `PageStats`.